pkgs/applications/version-management/git-and-tools/transcrypt/default.nix

```diff
@@ -1,20 +1,23 @@
-{ stdenv, fetchurl, git, openssl }:
+{ stdenv, fetchurl, git, makeWrapper, openssl }:
 
 stdenv.mkDerivation rec {
-  name = "transcrypt-0.9.7";
+  name = "transcrypt-0.9.9";
```

Comment:
Factoring out the version into its own attribute to be used in `name` and `url` would make updating slightly easier.